nav2_dwb_controller/dwb_core/src/dwb_local_planner.cpp

```diff
@@ -115,6 +118,7 @@ void DWBLocalPlanner::configure(
 
   node->get_parameter(dwb_plugin_name_ + ".prune_plan", prune_plan_);
   node->get_parameter(dwb_plugin_name_ + ".prune_distance", prune_distance_);
+  node->get_parameter(dwb_plugin_name_ + ".shorten_distance", shorten_distance_);
```

Comment:
While I don't have an issue with the concept, the name is a little non-descript. Any better options?

Reply:
I took inspiration from the related "shorten_transformed_plan" and "prune_distance" parameters. Any suggestion? future_plan_shortening_distance?

Reply:
`forward_prune_distance` perhaps

Reply:
Ok, done.